packages/ui/src/lib/button/Button.svelte

```diff
@@ -42,7 +42,11 @@ let {
   children,
 }: Props = $props();
 
-let iconType: string = $derived(isFontAwesomeIcon(icon) ? 'fa' : 'unknown');
+let iconType: string = $derived.by(() => {
+  if (isFontAwesomeIcon(icon)) return 'fa';
+  else if (typeof icon === 'string') return 'string';
```

Comment:
I think it might be confusing to have the 'string' type 
Maybe 'custom' ? Or 'font' or something else

but it's internal so can be changed any time